src/chi/components/data-table/data-table.scss

```diff
@@ -32,6 +32,14 @@ $bg-url: $bg-url + ' fill="%230262b9" fill-rule="evenodd"/%3E%3C/svg%3E';
   overflow-y: initial;
   width: 100%;
 
+  .-screen--only {
+    display: none;
+  }
+
+  .-print--only {
```

Comment:
-d-print--only

Reply:
Both classes should be defined as generic utility classes

Reply:
@evansito Why do we need these classes here if you define the utility classes in `display.scss`?